### PR TITLE
Fix ineffassign issues

### PR DESCRIPTION
**Pull request by mjtrangoni**
Hi @bergquist,

Related to #10381, I fixed all [ineffassign](https://github.com/gordonklaus/ineffassign) and added this check to CircleCI. See,

```
pkg/cmd/grafana-server/main.go:105:2:warning: ineffectual assignment to code (ineffassign)
pkg/components/dashdiffs/compare.go:143:2:warning: ineffectual assignment to err (ineffassign)
pkg/components/dynmap/dynmap_test.go:62:5:warning: ineffectual assignment to err (ineffassign)
pkg/components/dynmap/dynmap_test.go:110:2:warning: ineffectual assignment to s (ineffassign)
pkg/components/dynmap/dynmap_test.go:110:5:warning: ineffectual assignment to err (ineffassign)
pkg/components/dynmap/dynmap_test.go:150:12:warning: ineffectual assignment to err (ineffassign)
pkg/components/dynmap/dynmap_test.go:234:13:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/azureblobuploader_test.go:13:3:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/imguploader_test.go:21:12:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/imguploader_test.go:39:12:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/imguploader_test.go:57:12:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/imguploader_test.go:84:15:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/imguploader_test.go:109:12:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/imguploader_test.go:130:19:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/webdavuploader.go:43:11:warning: ineffectual assignment to err (ineffassign)
pkg/components/imguploader/webdavuploader.go:44:7:warning: ineffectual assignment to err (ineffassign)
pkg/log/file_test.go:34:4:warning: ineffectual assignment to err (ineffassign)
pkg/log/file_test.go:35:4:warning: ineffectual assignment to err (ineffassign)
pkg/services/alerting/notifiers/telegram.go:135:11:warning: ineffectual assignment to err (ineffassign)
pkg/services/alerting/notifiers/victorops.go:100:3:warning: ineffectual assignment to fields (ineffassign)
pkg/services/guardian/guardian_test.go:651:7:warning: ineffectual assignment to err (ineffassign)
pkg/services/provisioning/dashboards/file_reader.go:238:3:warning: ineffectual assignment to path (ineffassign)
pkg/services/sqlstore/annotation_test.go:258:5:warning: ineffectual assignment to err (ineffassign)
pkg/services/sqlstore/dashboard.go:80:2:warning: ineffectual assignment to affectedRows (ineffassign)
pkg/services/sqlstore/dashboard_acl_test.go:151:5:warning: ineffectual assignment to err (ineffassign)
pkg/services/sqlstore/playlist.go:24:5:warning: ineffectual assignment to err (ineffassign)
pkg/services/sqlstore/plugin_setting.go:50:11:warning: ineffectual assignment to err (ineffassign)
pkg/services/sqlstore/preferences.go:74:11:warning: ineffectual assignment to err (ineffassign)
pkg/services/sqlstore/team_test.go:76:5:warning: ineffectual assignment to err (ineffassign)
pkg/services/sqlstore/team_test.go:105:5:warning: ineffectual assignment to err (ineffassign)
pkg/services/sqlstore/user.go:171:2:warning: ineffectual assignment to user (ineffassign)
pkg/services/sqlstore/user.go:205:2:warning: ineffectual assignment to user (ineffassign)
pkg/services/sqlstore/migrations/migrations_test.go:30:4:warning: ineffectual assignment to has (ineffassign)
```

### CODE REVIEW

File: pkg/services/alerting/notifiers/victorops.go [hunk 1]
```diff
@@ -83,27 +83,6 @@ func (this *VictoropsNotifier) Notify(evalContext *alerting.EvalContext) error {
 		return nil
 	}
 
-	fields := make([]map[string]interface{}, 0)
```
Comment (bergquist) [inline pkg/services/alerting/notifiers/victorops.go]: not sure I understand why all those rows where removed?

Comment (mjtrangoni) [reply pkg/services/alerting/notifiers/victorops.go]: fields is not used at all, or I am wrong with this?

Comment (bergquist) [reply pkg/services/alerting/notifiers/victorops.go]: You're absolutely right.

**Comment by bergquist** [comment]
Thank you! :)

**Pull request merged**